Switching to bulma

torbensky opened the pull request.

torbinsky committed:
TW: switching to bulma

torbinsky committed:
TW: Cleanup

bridgetryan reviewed (changes requested):
Figure on main page should be smaller. Remove or complete the "get started now" button, since it currently doesn't link anywhere.

brimmus committed:
Addressed PR needs

- made image smaller
- centered image within figure
- added top margin to figure
- made button go to app
